.github/workflows/run_tests.yml

```diff
@@ -66,17 +66,37 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: s-weigand/setup-conda@v1
+        with:
+          python-version: 3.7
       - uses: actions/setup-java@v1
         with:
           java-version: 1.8
       - uses: actions/setup-node@v1
       - name: Run JS linter
         run: |
+          cd "${GITHUB_WORKSPACE}"
+          pip install -r requirements.txt
+          conda install -c rdkit rdkit
+          python setup.py install
+          pip install flask
+          cd "${GITHUB_WORKSPACE}/editor"
+          npm install google-protobuf puppeteer
+          wget https://storage.googleapis.com/ord-editor-test/editor_test_protobuf_1dae8fdd.tar
+          tar -xzf editor_test_protobuf_1dae8fdd.tar
+          wget https://github.com/google/closure-library/archive/v20200517.tar.gz
+          tar -xzf v20200517.tar.gz
+          git clone https://github.com/Open-Reaction-Database/ketcher.git
+          make
+
           cd "${GITHUB_WORKSPACE}"
           # See https://github.com/google/closure-compiler/wiki/Lint-checks.
           npm install google-closure-compiler
-          java -jar node_modules/google-closure-compiler-java/compiler.jar \
-            --jscomp_warning=lintChecks editor/js/*.js
+
+          # Build and lint, but supress warnings on anything but our code.
+          java -jar node_modules/google-closure-compiler-java/compiler.jar --js 'editor/closure-library-20200517/**.js' --js 'editor/protobuf/js/**.js' --js 'editor/gen/js/proto/ord/**.js'  --js editor/js/*.js --checks_only --dependency_mode PRUNE --entry_point ord.reaction --jscomp_warning=lintChecks --hide_warnings_for=editor/closure-library-20200517 --hide_warnings_for=editor/closure-library-20200517 --hide_warnings_for=editor/protobuf/js --hide_warnings_for=gen/js/proto/ord
+
+          java -jar node_modules/google-closure-compiler-java/compiler.jar --js 'editor/closure-library-20200517/**.js' --js 'editor/protobuf/js/**.js' --js 'editor/gen/js/proto/ord/**.js'  --js editor/js/*.js --checks_only --dependency_mode PRUNE --entry_point ord.dataset --jscomp_warning=lintChecks --hide_warnings_for=editor/closure-library-20200517 --hide_warnings_for=editor/closure-library-20200517 --hide_warnings_for=editor/protobuf/js --hide_warnings_for=gen/js/proto/ord
```

Comment:
Is there a reason this is run twice?

Reply:
Looks like they have different `--entry_point` flags.